packages/common/src/ui/components/modals/ListSearch/ListSearch.stories.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { Grid, Button } from '@mui/material';
+import { Story } from '@storybook/react';
+import { ListSearch } from './ListSearch';
+
+export default {
+  title: 'Modals/ListSearch',
+  component: ListSearch,
+};
+
+const options = Array.from({ length: 100 }).map((_, i) => ({
+  name: `${i}`,
+}));
```

Comment:
Was thinking to maybe look at exporting some functions from `mock-server` to generate random data. Will also hook up a new story which would use some async data, so will revisit this there